front/components/ideaDetail/IdeaCardSkeleton.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="idea-card-skeleton">
```

Comment:
maybe we can use https://vuetifyjs.com/en/components/skeleton-loaders/ ?

Reply:
Yes, didn't know that it has this components, I'll redo

Reply:
Done